src/compas_rhino/install.py

```diff
@@ -106,11 +115,40 @@ def install(version=None, packages=None):
         if status != 'OK':
             exit_code = -1
 
+    if len(installed_packages):
+        print()
+        print('Running post-installation steps...')
+        print()
+        _run_post_execution_steps(after_rhino_install(installed_packages))
+
     print('\nCompleted.')
     if exit_code != 0:
         sys.exit(exit_code)
 
 
+def _run_post_execution_steps(steps_generator):
+    post_execution_errors = []
+    for result in steps_generator:
+        if isinstance(result, Exception):
+            post_execution_errors.append(result)
+            continue
+
+        for item in result:
+            try:
+                package, message, success = item
+                status = 'OK' if success else 'ERROR'
+                print('   {} {}: {}'.format(package.ljust(20), status, message))
+            except ValueError:
+                post_execution_errors.append(ValueError('Step successful, but result is wrongly formatted: {}'.format(str(result))))
```

Comment:
and here, should `result` be `item`?

Reply:
ups! https://github.com/compas-dev/compas/pull/770/commits/6440b4d499e3b9e09552f09165a80090cfa40e8b